packages/framework-core/src/services/graphql/graphql-query-generator.ts

```diff
@@ -83,6 +87,61 @@ export class GraphQLQueryGenerator {
     return queries
   }
 
+  private generateEventQueries(): GraphQLFieldConfigMap<unknown, GraphQLResolverContext> {
+    const eventQueryResponse = this.buildEventQueryResponse()
+    return {
+      eventsByEntity: {
+        type: eventQueryResponse,
+        args: {
+          entity: {
+            type: new GraphQLNonNull(this.buildGraphqlSimpleEnumFor('EntityType', Object.keys(this.config.entities))),
```

Comment:
Thanks to this GraphQL enum inferred from the entities in the config, it will be impossible for the user to specify a non-existing entity name in the query. They will get a GraphQL validation error. One of the beauty of GraphQL